tensorflow_probability/examples/BUILD

```diff
@@ -27,6 +27,45 @@ package(
 
 exports_files(["LICENSE"])
 
+py_binary(
+    name = "keras_bayesian_neural_network",
```

Comment:
Instead of adding new targets, can you go ahead and replace the old ones? (i.e., just overwrite `bayesian_neural_network.py` and `logistic_regression.py` with your new versions). I don't think there's a lot of value in us keeping around out-of-date examples (debatably there's negative value :).